dkarlovi/symfony-flex-docker/0.1/Makefile

```diff
@@ -0,0 +1,35 @@
+docker-start:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.$(APP_ENV).yaml up -d
+.PHONY: docker-up
+
+docker-stop:
+	docker-compose down
+.PHONY: docker-down
+
+docker-start-dev:
```

Comment:
You can already do 

     make docker-start APP_ENV=prod

I just feel 

    make docker-start-prod

is easier to type. But you might be right, the additional targets might not be significant enough (or used frequently enough) to have all these shortcuts in.